### ISSUE TITLE
AttributeError: module 'shlex' has no attribute 'join'

### ISSUE THREAD

**Issue by TLMcNulty** [k4yt3x/rnnoise-pulseaudio-control#1]
Opening this while working on debugging. 
```
pulseaudio 12.2
19:28 $ lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 10 (buster)
Release:	10
Codename:	buster
```

**Comment by k4yt3x** [comment]
Python version's too old. This requires Python >= 3.8. If  you don't want to install Python 3.8, you can manually change all the `shlex.join(command)`s to `" ".join(command)`.

![image](https://user-images.githubusercontent.com/21986859/121763878-553ce680-cb0d-11eb-9033-5ddcdfd472d4.png)

**Comment by TLMcNulty** [comment]
Yep, been working through an upgrade. 

Trying to install version 3.9.5 but can't install the required modules for the new version due to openssl linking errors. Going to continue working. This can be closed resolution Python >= 3.8.
